<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameMain</class>
 <widget class="QDialog" name="gameMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="panstwoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Państwo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="miastoLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Miasto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="zwierzeLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Zwierze</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="roslinaLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Roślina</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="imieLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Imię</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="wodyLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Wody</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="przedmiotLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Przedmiot</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="slawnaOsobaLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sławna osoba</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="slawnaOsobaTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="miastoTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="wodyTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="przedmiotTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>530</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="imieTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="roslinaTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="zwierzeTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="panstwoTextEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="panstwoCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="roslinaCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="slawnaOsobaCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>610</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="zwierzeCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="miastoCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="imieCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="przedmiotCheckEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="wodyCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
